### PR TITLE
LPS-129598 Migrate selection modals in fragment modules

### PR DESCRIPTION
**Pull request by jonmak08**
Hey @liferay-frontend,

Attached is an update for http://issues.liferay.com/browse/LPS-129598.

Test Plan:

1. Login to Liferay
2. Click on the **Product Navigation Menu**
3. Go to **Design** -> **Fragments**
4. Click the `+` to add a new fragment collection
5. Click the `+` on the fragment collection to add a new fragment
6. Click on the ellipsis of the new fragment
7. Select **Move**
fragment

Please let me know if you have any questions. Thanks!

### CODE REVIEW

**Review by kresimir-coko — APPROVED** [review]

**Review by markocikos — CHANGES_REQUESTED** [review]
@jonmak08 The changes in this PR look good! :+1: 

But, there is a use of a [selection-related custom event still in a fragment module](https://github.com/liferay/liferay-portal/blob/master/modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/view_fragment_collections.jsp#L68-L81). We should remove it. Note that this is one of the three formats of returning data that we don't support yet. See https://github.com/liferay-frontend/liferay-portal/pull/933#issuecomment-808360465. We need to figure out a best way to distinguish between them and [set](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js#L430-L434) the correct one.

**Comment by jonmak08** [comment]
Hey @markocikos,

I updated this PR based on the changes you sent at https://github.com/liferay-frontend/liferay-portal/pull/951.

Please let me know if you have any questions. Thanks!

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/actions.js [hunk 1]
```diff
@@ -25,28 +25,35 @@ export const ACTIONS = {
 			Liferay.Language.get('delete-collection'),
 			viewDeleteFragmentCollectionsURL,
 			(selectedItems) => {
-				const fragmentCollectionsForm = document.getElementById(
-					`${portletNamespace}fragmentCollectionsFm`
-				);
+				if (selectedItems.length) {
```
Comment (markocikos) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/actions.js:28]: I tweaked my code to have a null check and an early return, see https://github.com/pei-jung/liferay-portal/pull/824/commits/c3bfd84d468dddef36403ce175641fd8dec4f48f
- Although unlikely, it is possible that `selectedItems` is unset, if devs opt to use custom event.
- Early return is just to buy space, as Brian would say

**Review by markocikos — APPROVED** [review]
Minor SF inline. Other than that, we can ship this as soon as https://github.com/pei-jung/liferay-portal/pull/824 is merged.

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/actions.js (hunk 1 above)
Comment (jonmak08) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/actions.js:28]: Sounds good!  Thanks!  I will update my code as necessary.

Comment (jonmak08) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/actions.js]: Fixed at https://github.com/liferay-frontend/liferay-portal/pull/948/commits/cf4ea362db69088c0061072bbc85b7e67bc5ba75#diff-d144a92ad201a6b94c6a6cf6c91984dd47a7f4a02ada1724e056497167b90c33R28

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

**Comment by jonmak08** [comment]
ci:test:relevant

**Comment by jonmak08** [comment]
New Pull Request: https://github.com/liferay-echo/liferay-portal/pull/4428

**Pull request closed without merging**